prow/cmd/entrypoint/main.go

```diff
@@ -53,6 +55,7 @@ func gatherOptions() options {
 	o := options{}
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 	o.wrapperOptions = wrapper.BindOptions(fs)
+	fs.DurationVar(&o.timeout, "timeout", 5*time.Hour, "Timeout for the test command.")
```

Comment:
we won't be able to switch until this is properly tunable from prow config, scalability has 12+ hour jobs

Reply:
I'd expect the first implementation of `prow` config that feeds `plank` to expose this. The first prototypes won't thread through config but will just explicitly create a complicated `PodSpec`, I think?